gui/RemoveItem.java

```diff
@@ -216,33 +243,42 @@ public void setup() {
 		// The students on the Roster List for this class
 		ArrayList<ListEntry> rlist = new ArrayList<ListEntry>();
 
-		for (String s : AdminUI.roster) {
+		for (String s : MentorUI.roster) {
```

Comment:
nah don't worry about it, like i said this will change to a manager because of the multiple rosters possible